clients/client-web/package.json

```diff
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "crypto-js": "^3.1.9-1",
-    "hawk": "^7.0.7",
+    "hawk": "^8.0.0",
```

Comment:
This part is problematic, because Hawk dropped its browser compatibility mode around this version.  So, I think we should freeze this at 7.0.7.  The other uses of this library are fine.